test/tests/functional/pbs_trillion_jobid.py

```diff
@@ -0,0 +1,497 @@
+            runas=ROOT_USER,
+            expect=True)
+        self.server.log_match('svr_max_job_sequence_id reverting back '
+                              'to default val %d' % (9999999),
+                              starttime=self.server.ctime)
+
+    def test_max_job_sequence_id_values(self):
+        """
+        Test to check valid/invalid values for the
+        max_job_sequence_id server attribute
+        """
+        # Invalid Values
+        invalid_values = [-9999999, '*456879846',
+                          23545.45, 'ajndd', '**45', 'asgh456']
+        for val in invalid_values:
+            try:
+                seq_id = {ATTR_max_job_sequence_id: val}
+                self.server.manager(
+                    MGR_CMD_SET, SERVER, seq_id, runas=ROOT_USER)
+            except PbsManagerError as e:
+                self.assertTrue(
+                    'Illegal attribute or resource value' in e.msg[0])
+        # Less than or Greater than the attribute limit
+        min_max_values = [120515, 999999, 1234567891234, 9999999999999]
+        for val in min_max_values:
+            try:
+                seq_id = {ATTR_max_job_sequence_id: val}
+                self.server.manager(
+                    MGR_CMD_SET, SERVER, seq_id, runas=ROOT_USER)
+            except PbsManagerError as e:
+                self.assertTrue('Cannot set max_job_sequence_id < 9999999, '
+                                'or > 999999999999' in e.msg[0])
+        # Valid values
+        valid_values = [9999999, 123456789, 100000000000, 999999999999]
+        for val in valid_values:
+            seq_id = {ATTR_max_job_sequence_id: val}
+            self.server.manager(
+                MGR_CMD_SET,
+                SERVER,
+                seq_id,
+                runas=ROOT_USER,
+                expect=True)
+            self.server.expect(SERVER, seq_id)
+
+    def test_max_job_sequence_id_wrap(self):
+        """
+        Test to check the jobid's/resvid's are wrapping it to zero or not,
+        after reaching to the given limit
+        """
+        # Check default limit(9999999) and wrap it 0
+        a = {'resources_available.ncpus': 10}
+        self.server.manager(MGR_CMD_SET, NODE, a, self.mom.shortname)
+        self.submit_job(sleep=2)
+        self.submit_job(sleep=2, lower=1, upper=2)
+        self.submit_resv()
+        time.sleep(6)
+        sv_jobidnumber = 9999999  # default
+        self.set_svr_sv_jobidnumber(sv_jobidnumber)
+        jid_3 = self.submit_job(sleep=2)
+        self.assertEqual(jid_3.split('.')[0], '%s' % (sv_jobidnumber))
+        jid_4 = self.submit_job(sleep=2, lower=1, upper=2)
+        self.assertEqual(jid_4.split('.')[0], '0[]')  # wrap it
+        rid_2 = self.submit_resv()
+        self.assertEqual(rid_2.split('.')[0], 'R1')
+
+        # Check max limit (999999999999) and wrap it 0
+        sv_jobidnumber = 999999999999  # max limit
+        seq_id = {ATTR_max_job_sequence_id: '%d' % (sv_jobidnumber)}
```

Comment:
no need of %d

Reply:
@visheshh, using sv_jobidnumber at 3 other places also that's why used like this.

Reply:
@Bhagat-Rajput Just remove %d and directly use sv_jobidnumber